src/aioswitcher/bridge.py

```diff
@@ -137,8 +147,31 @@ def _parse_device_from_datagram(
                     parser.get_ip_type2(),
                     parser.get_mac(),
                     parser.get_name(),
-                    parser.get_shutter_position(),
-                    parser.get_shutter_direction(),
+                    device_type.token_needed,
```

Comment:
I'm wondering why we have diverged from the "parser" approach.

I also noticed this in rows 170 and 189.

Reply:
@TomerFi Because these parts are getting values from device_type and not from the packet message so we don't need to use a "parser".
I can make a function that returns the same and we use "parser." but I don't see any benefit.